corehq/apps/enterprise/static/enterprise/js/project_dashboard.js

```diff
@@ -175,7 +175,11 @@ hqDefine("enterprise/js/project_dashboard", [
         const requestParams = {
             url: initialPageData.reverse("enterprise_dashboard_total", slug),
             success: function (data) {
-                $display.html(Number(data.total).toLocaleString());
+                if (typeof data.total === 'number') {
+                    $display.html(Number(data.total).toLocaleString());
+                } else {
+                    $display.html(data.total);
+                }
```

Comment:
If this is intended to support percentages, do we need to be concerned about locale formatting (i.e. using ',' for a decimale separator) for those as well?

Reply:
Cherry picked your commit in PR #35480 in 1b67a86b55f8e73d3145e7e64c42be7a8c2f7d49